assets/css/learning-mode.scss

```diff
@@ -3,9 +3,6 @@
 @import 'sensei-course-theme/video-container';
 @import 'sensei-course-theme/ui-blocks';
 @import 'sensei-course-theme/fixed-sidebar';
-@import 'sensei-course-theme/theme-fixes';
-
-@import 'sensei-course-theme/buttons';
```

Comment:
Note that `learning-mode.4-0-2.scss` also have these imports. Only files in the `4-0-2/` folder are specific to that version, so I think we won't need a separate `learning-mode.4-0-2.compat.scss`.

Also we can decide to not support the `4.0.2` template version if it's too much trouble, since those would mainly be used by existing users who will have to re-create their customizations anyway if they change themes.

Reply:
Thanks for pointing it out @yscik, I've removed import from `learning-mode.4-0-2.scss` as well here and added a conditional section to load the default `learning-mode-compat.scss` as we don't have a separate one for 4.0.2.